### ISSUE TITLE
Player Spawns with head on the floor.

### ISSUE THREAD

**Issue by atahan-git** [ValveSoftware/steamvr_unity_plugin#935]
![image](https://user-images.githubusercontent.com/14096462/113247238-92a71b80-92f5-11eb-8a3b-132da7b3f8ae.png)

I received this when trying to submit my application. During my own testing everything worked perfectly fine. Is there any way I can somehow debug this?

I didn't change anything related to head height of the SteamVR prefabs, so I have no idea why this is happening.

**Comment by zite** [comment]
You received this from Valve?

**Comment by atahan-git** [comment]
Yeah I did receive this from Valve. It seems like the problem was that my play mode was set to seated, which automatically starts your head on the floor. However this is not the case with the oculus quest. With a quest, whether you are seated or not it always tracks you at the head level, not at the origin.
